pkgs/development/libraries/openal-soft/default.nix

```diff
@@ -25,6 +25,13 @@ stdenv.mkDerivation rec {
       --replace "@OUT@" $out
   '';
 
+  postInstall = ''
+    # the pc file contains the nix store path twice
+    substituteInPlace $out/lib/pkgconfig/openal.pc \
+      --replace "libdir=\''${exec_prefix}/$out/lib" "libdir=\''${exec_prefix}/lib" \
+      --replace "includedir=\''${prefix}/$out/include" "includedir=\''${prefix}/include"
```

Comment:
I looked upstream and don't understand why we end up with this broken file:

* https://github.com/kcat/openal-soft/blob/master/openal.pc.in
* https://github.com/kcat/openal-soft/blob/master/CMakeLists.txt#L1262-L1266

Reply:
Ah, we need to bump the version, the pkg-config file was fixed upstream in v1.2.21. Latest version is v1.2.22.

Reply:
done.